pygmt/src/colorbar.py

```diff
@@ -66,26 +69,26 @@ def colorbar(self, **kwargs):
         changed by appending +j followed by a 2-char justification code
         *justify*.
     box : bool or str
-        ``[+cclearances][+gfill][+i[[gap/]pen]][+p[pen]][+r[radius]]
-        [+s[[dx/dy/][shade]]]``. If set to True, draws a rectangular
-        border around the color scale. Alternatively, specify a different
-        pen with +ppen. Add +gfill to fill the scale panel [no fill].
-        Append +cclearance where clearance is either gap, xgap/ygap, or
-        lgap/rgap/bgap/tgap where these items are uniform, separate in x-
-        and y-direction, or individual side spacings between scale and
-        border. Append +i to draw a secondary, inner border as well. We use
-        a uniform gap between borders of 2p and the MAP_DEFAULTS_PEN unless
-        other values are specified. Append +r to draw rounded rectangular
-        borders instead, with a 6p corner radius. You can override this
-        radius by appending another value. Finally, append +s to draw an
+        [**+c**\ *clearances*][**+g**\ *fill*][**+i**\ [[*gap*/]\ *pen*]]
+        [**+p**\ [*pen*]][**+r**\ [*radius*]][**+s**\ [[*dx*/*dy*/][*shade*]]].
+        If set to True, draws a rectangular border around the color scale.
+        Alternatively, specify a different pen with +ppen. Add +gfill to fill
+        the scale panel [no fill]. Append +cclearance where clearance is either
+        gap, xgap/ygap, or lgap/rgap/bgap/tgap where these items are uniform,
+        separate in x- and y-direction, or individual side spacings between
+        scale and border. Append +i to draw a secondary, inner border as well.
+        We use a uniform gap between borders of 2p and the MAP_DEFAULTS_PEN
+        unless other values are specified. Append +r to draw rounded
+        rectangular borders instead, with a 6p corner radius. You can override
+        this radius by appending another value. Finally, append +s to draw an
         offset background shaded region. Here, dx/dy indicates the shift
         relative to the foreground frame [4p/-4p] and shade sets the fill
         style to use for shading [gray50].
     truncate : list or str
-        ``zlo/zhi`` Truncate the incoming CPT so that the lowest and
-        highest z-levels are to zlo and zhi. If one of these equal NaN then
-        we leave that end of the CPT alone. The truncation takes place
-        before the plotting.
+        *zlo*/*zhi*.
+        Truncate the incoming CPT so that the lowest and highest z-levels are
+        to zlo and zhi. If one of these equal NaN then we leave that end of the
+        CPT alone. The truncation takes place before the plotting.
```

Comment:
```suggestion
        *zlo*/*zhi*.
        Truncate the incoming CPT so that the lowest and highest z-levels are
        to *zlo* and *zhi*. If one of these equal NaN then we leave that end of the
        CPT alone. The truncation takes place before the plotting.
```